src/pages/contacts/_address.vue

```diff
@@ -72,7 +72,7 @@ export default Vue.extend({
   },
   computed: {
     computedReturnLink(): Route | string {
-      return this.fromRoute && this.fromRoute.fullPath !== this.$route.fullPath && this.fromRoute.path !== "/transaction/transfer" ? this.fromRoute : "/contacts";
+      return this.fromRoute && this.fromRoute.fullPath !== this.$route.fullPath && !this.fromRoute.path.includes("/transaction/") ? this.fromRoute : "/contacts";
```

Comment:
`this.fromRoute && this.fromRoute.fullPath` can be simplified w/t the ts `?` or `!`.
`!this.fromRoute.path.includes("/transaction/")` IMO should be replaced w/t `.startsWith`